backend/api/serializers/CreditTrade.py

```diff
@@ -402,7 +402,7 @@ def get_actions(self, obj):
 
         # If the user doesn't have any roles assigned, treat as though the user
         # doesn't have available permissions
-        if request.user.role is None:
+        if request.user.roles is None:
```

Comment:
Again, I don't think this can happen. Should this check go away or change to a permission check if they need a specific permission to continue?